test/ApplicationInsightsJavaScriptSnippetTest/ApplicationInsightsJavaScriptSnippetTest.cs

```diff
@@ -78,8 +78,14 @@ public ApplicationInsightsJavaScriptSnippetTest(ITestOutputHelper output) : base
 
                     var validator = new Validator(httpClient, httpClientHandler, logger, deploymentResult);
 
-                    logger.LogInformation("Verifying home page");
-                    await validator.VerifyScriptCheckPage(response);
+                    logger.LogInformation("Verifying layout page");
+                    await validator.VerifyLayoutPage(response);
+
+                    logger.LogInformation("Verifying layout page before script");
+                    await validator.VerifyLayoutPageBeforeScript(response);
+
```

Comment:
I think just put a comment here saying that we don't validate the script itself because it could change, but instead we verify that it did *something*. (I want to make sure a future person doesn't inadvertently "fix" this test by having it verify too much.)